gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -612,7 +612,7 @@ message Alert {
     NO_SERVICE = 1;
     REDUCED_SERVICE = 2;
 
-    // We don't care about INsignificant delays: they are hard to detect, have
```

Comment:
I don't think this is a typo. It's meant to highlight the "in" in insignificant.